packages/trackers-preview/src/components/trackers-preview.js

```diff
@@ -9,13 +9,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0
  */
 
-import { html, define } from '/hybrids.js';
-
-import { externalLink, close } from '../../../../ui/src/components/icons.js';
-import '../../../../ui/src/components/panel-header.js';
-import '../../../../ui/src/components/wtm-stats.js';
-
-import { t } from '../../../../ui/src/i18n.js';
+import { html, define } from 'hybrids';
+import '@ghostery/ui';
```

Comment:
It will be redundant, as it must be done in one proper way. It also allows to add appropriate tags to the document much simpler, and use custom elements, without additional scripts. For example, the material web components from Google are built in that way - https://github.com/material-components/material-web.

```html
<script src="https://esm.sh/@ghostery/ui" type="module"></script>
<ui-header></ui-header>
```